include/common/Exception.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <Poco/Exception.h>
+#include <exception>
+#include <string>
+
+namespace pingcap
+{
+
+const int MismatchClusterIDCode = 1;
+const int GRPCErrorCode = 2;
+const int InitClusterIDFailed = 3;
+const int UpdatePDLeaderFailed = 4;
+const int TimeoutError = 5;
+const int RegionUnavailable = 6;
+const int LogicalError = 7;
+const int LockError = 8;
+const int LeanerUnavailable = 9;
+const int StoreNotReady = 10;
+const int RaftEntryTooLarge = 11;
+const int ServerIsBusy = 12;
+const int LeaderNotMatch = 13;
```

Comment:
done

Reply:
Well, I don't think using `enum` or `int` is a good idea. `enum class` is more safety here.

Reply:
But Poco::Exception only accepts `int` for ErrorCode

Reply:
Well, I think it's ok to use enum, but personally, I don't like implicit type casting. I'm more likely to use styles like:

```
enum class xxx: int {
    MismatchClusterIDCode = 1,
    GRPCErrorCode
}
```

and when we using it, we can:

```
static_cast<int>(xxx);
```
or just:
```
static_cast<typename std::underlying_type<E>::type>(e);
```

Reply:
If you think `enum` is ok, you can just ignore my advise and resolve this conversation.

Reply:
This makes code more complex. In my opinion, to use it as int, using `enum` rather than `enum class` is ok. This kind of implicit type conversion doesn't harm.